systems/plants/@RigidBodyManipulator/contactConstraintDerivatives.m

```diff
@@ -1,13 +1,29 @@
-function [n, D, dn, dD] = contactConstraintDerivatives(obj, kinsol, idxA, idxB, xA, xB, normal, d, use_mex)
- compute_second_derivative = nargout > 2;
- 
-  if(use_mex)
+function [d, n, D, dn, dD] = contactConstraintDerivatives(obj, use_mex, normal, kinsol, idxA, idxB, xA, xB)
```

Comment:
If we move use_mex back to the end, the call from contactConstraints.m that just gets the tangents would have to pass in a bunch of unused arguments to get to it.  Maybe the best solution is to pull the mex-checking logic back out into contactConstraints.m?